README.md

```
@@ -1,119 +1,66 @@
-# Hymn Broadcast Console + Overlay Server
+# SDA Hymnal Desktop
+### Professional Hymn Broadcast Console & Overlay Server
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)